Documentation/ApiOverview/DirectoryStructure/Index.rst

```diff
@@ -27,6 +27,24 @@ the paths within PHP code.
   - :Directory,20: Directory
     :Description,80: Description
 
+  - :Directory: :file:`_assets/`
+    :Description:
+      This directory includes symlinks to resources of extensions, as consequence
+      of this and further structure changes the folder :file:`typo3conf/ext/` is
+      not created or used anymore.  
+      So all files like CSS, JavaScript, Icons, Fonts, Images, etc. of extensions 
+      are not linked anymore directly to the extension folders but to the directory
+      :file:`_assets/`.
+      
+      .. note::
+        This directory :file:`_assets/` and the related changes depend on the
+        system-extension `composer_installers` in version 4+.
```

Comment:
It is not a system extension, it is a composer dependency and only applies to composer installations. And it should be "typo3/cms-composer-installers"

Reply:
Composer never needs the extension but TYPO3, so it's no composer dependency